### ISSUE TITLE
Stable release

### ISSUE THREAD

**Issue by Inscure** [tabler/tabler#1972]
### Discussed in https://github.com/tabler/tabler/discussions/1971

<div type='discussions-op-text'>

<sup>Originally posted by **Inscure** October  3, 2024</sup>
Hello,

First of all, thank you for the great work on Tabler! We are very interested in using the tool in our projects. However, we have noticed that only beta versions are available at the moment.

Our company's policy requires us to use stable releases for production environments. Could you kindly let us know when you plan to release a stable version of Tabler?

We would love to adopt it as soon as it's available.

Thank you in advance for your response!</div>

**Comment by jonbk** [comment]
The distinction between "beta" and "stable" can be somewhat ambiguous, especially for a frontend library focused primarily on graphical rendering (HTML/CSS). In practice, Tabler is already very stable and doesn’t present any particular security concerns, given that it’s largely composed of HTML and CSS. As for the JavaScript, it relies on well-established third-party libraries like Bootstrap and ApexCharts, which are widely used and trusted in the industry.

Personally, I've been using Tabler in a production project for the past two years without any issues. I imagine the main maintainer will move to a 1.0 version when they feel the library's component offerings are sufficiently comprehensive.

IMHO, the "beta" label doesn't imply that Tabler isn’t production-ready. You can safely adopt it for production use.

**Comment by ARehmanMahi** [comment]
Beta labeling is very weird in this scenario. 
Almost all the projects that use tbler are in production.
If by beta the devs feel it is incomplete regarding components then, that is not a correct analogy for labeling it beta.

Especially when they have v1.0 build

**Comment by Kerrick** [comment]
I agree -- I'd love to see the current version (1.0.0-beta24) simply be re-released as 1.0.0, with future non-breaking updates being released as patch (1.0.1) and/or minor (1.1.0) versions. Tabler has seemed stable enough to earn that non-beta label for months. :shipit:

**Comment by mholt** [comment]
Looks like wish granted 🌟

**Comment by codecalm** [comment]
yup, we've released version 1.0 today 🙂

**Comment by mholt** [comment]
Congrats @codecalm !
